lib/iris/tests/integration/netcdf/test__chunk_control.py

```diff
@@ -0,0 +1,161 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""Integration tests for loading and saving netcdf files."""
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests  # isort:skip
+
+from pathlib import Path
+import shutil
+import tempfile
+from unittest.mock import ANY, Mock
+
+import dask
+from dask import distributed
+from numpy import dtype
+import pytest
+
+import iris
+from iris import _lazy_data
+from iris.fileformats.netcdf import loader
+from iris.fileformats.netcdf.loader import CHUNK_CONTROL
+import iris.tests.stock as istk
+
+
+@pytest.fixture()
+def create_cube(tmp_filepath):
+    cube = istk.simple_4d_with_hybrid_height()
+    cube_varname = "my_var"
+    sigma_varname = "my_sigma"
+    cube.var_name = cube_varname
+    cube.coord("sigma").var_name = sigma_varname
+    cube.coord("sigma").guess_bounds()
+    iris.save(cube, tmp_filepath)
+    yield cube_varname, sigma_varname
+
+
+@pytest.fixture
+def create_file_cube(tmp_filepath):
+    iris.save(istk.simple_3d(), tmp_filepath, chunksizes=(1, 3, 4))
+    yield None
+
+
+@pytest.fixture
+def tmp_filepath():
+    tmp_dir = Path(tempfile.mkdtemp())
+    tmp_path = tmp_dir / "tmp.nc"
+    yield tmp_path
+    shutil.rmtree(tmp_dir)
+
+
+@pytest.fixture(autouse=True)
+def remove_min_bytes():
+    old_min_bytes = loader._LAZYVAR_MIN_BYTES
+    loader._LAZYVAR_MIN_BYTES = 0
+    yield None
+    loader._LAZYVAR_MIN_BYTES = old_min_bytes
+
+
+def test_default(tmp_filepath, create_cube):
+    cube = iris.load_cube(tmp_filepath, create_cube[0])
```

Comment:
Using `iris.load_cube()` within a test makes us vulnerable to a lot of unrelated breakages - it's a big call stack.

These are effectively unit tests, but because they are testing loading we can't get them to be super low level. We can get them a bit lower, though: please could you use `iris.fileformats.netcdf.loader.load_cubes()` instead of `iris.load_cube()`? This bypasses a lot of the call stack.